tests/integration_tests/kafka_simple_handle_key_only_avro/run.sh

```
@@ -32,7 +32,22 @@ function run() {
 	cdc_cli_changefeed create --start-ts=$start_ts --sink-uri="$SINK_URI" -c ${changefeed_id} --config="$CUR/conf/changefeed.toml"
 	run_sql_file $CUR/data/ddl.sql ${UP_TIDB_HOST} ${UP_TIDB_PORT}
 
-	sleep 5
+	# Ensure the DDL is sent before lowering max-message-bytes. Handle-key-only only applies to DML.
+	ddl_done_tso=$(run_cdc_cli_tso_query ${UP_PD_HOST_1} ${UP_PD_PORT_1})
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)